docs/standard/microservices-architecture/secure-net-microservices-web-applications/authorization-net-microservices-web-applications.md

```diff
@@ -106,21 +106,17 @@ If the user meets the requirement, a call to context.Succeed will indicate that
 
 In addition to registering custom policy requirements with AddPolicy calls, you also need to register custom requirement handlers via Dependency Injection (services.AddTransient&lt;IAuthorizationHandler, MinimumAgeHandler&gt;()).
 
-An example of a custom authorization requirement and handler for checking a user’s age (based on a DateOfBirth claim) is available in the ASP.NET Core [authorization documentation](https://docs.asp.net/en/latest/security/authorization/policies.html).
+An example of a custom authorization requirement and handler for checking a user's age (based on a DateOfBirth claim) is available in the ASP.NET Core [authorization documentation](/aspnet/core/security/authorization/policies).
 
 ## Additional resources
 
--   **ASP.NET Core Authentication**
-    [*https://docs.microsoft.com/aspnet/core/security/authentication/identity*](https://docs.microsoft.com/aspnet/core/security/authentication/identity)
+-   [Introduction to Identity on ASP.NET Core](/aspnet/core/security/authentication/identity)
```

Comment:
I wasn't touching on these links on purpose. @BillWagner what do you think? Since this is also source for the microservices book, they don't follow the same pattern.